crates/subspace-node/src/chain_spec.rs

```diff
@@ -71,65 +70,25 @@ struct GenesisParams {
     confirmation_depth_k: u32,
 }
 
-pub fn gemini_3c() -> Result<ConsensusChainSpec<GenesisConfig, SystemDomainGenesisConfig>, String> {
-    Err("Wrong release for Gemini 3c. Use the release prefixed with `gemini-3c`".to_string())
-}
-
-pub fn gemini_3c_compiled(
+pub fn gemini_3d_compiled(
 ) -> Result<ConsensusChainSpec<GenesisConfig, SystemDomainGenesisConfig>, String> {
     Ok(ConsensusChainSpec::from_genesis(
         // Name
-        "Subspace Gemini 3c",
+        "Subspace Gemini 3d",
         // ID
-        "subspace_gemini_3c",
-        ChainType::Custom("Subspace Gemini 3c".to_string()),
+        "subspace_gemini_3d",
+        ChainType::Custom("Subspace Gemini 3d".to_string()),
         || {
             let sudo_account =
-                AccountId::from_ss58check("5CXTmJEusve5ixyJufqHThmy4qUrrm6FyLCR7QfE4bbyMTNC")
+                AccountId::from_ss58check("5CZy4hcmaVZUMZLfB41v1eAKvtZ8W7axeWuDvwjhjPwfhAqt")
                     .expect("Wrong root account address");
 
-            let mut balances = vec![(sudo_account.clone(), 1_000 * SSC)];
-            let vesting_schedules = TOKEN_GRANTS
```

Comment:
Why did you delete this vesting code? It should remain in every network revision.